### DISCUSSION TITLE
AIC for ARIMA model

### DISCUSSION THREAD

**Discussion by theforecastnoob** [Nixtla/statsforecast#922] [Q&A] [answered]
Do you plan to make it possible to obtain e.g. AIC from an ARIMA? I know AutoArima provides this, but cannot find information about this regarding the standard ARIMA.

**Comment by jmoralez** [comment, marked as answer]
The criterions are also included in the arima model, e.g.

```python
from statsforecast.models import ARIMA
from statsforecast.utils import AirPassengers

model = ARIMA(season_length=12).fit(AirPassengers)
model.model_['aic'], model.model_['bic']
```
